components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/policy/store/PolicyStoreManager.java

```diff
@@ -39,21 +40,17 @@
  */
 public class PolicyStoreManager {
 
-    private PolicyStoreManageModule policyStoreStore = null;
-    private PolicyDataStore policyDataStore = null;
+    private final PolicyDAO policyStore;
 
-    private static Log log = LogFactory.getLog(PolicyStoreManager.class);
```

Comment:
Why we removed the log here, I think its better to add some debug logs WDYT?

Reply:
It was removed since it is not used. This file doesn't handle any exceptions, and logs have been added at lower places where the exceptions are thrown.